docs/sql-ref-syntax-qry-select.md

```diff
@@ -41,14 +41,23 @@ select_statement [ { UNION | INTERSECT | EXCEPT } [ ALL | DISTINCT ] select_stat
 
 While `select_statement` is defined as
 ```sql
-SELECT [ hints , ... ] [ ALL | DISTINCT ] { [ [ named_expression | regex_column_names | star ] [ , ... ] | TRANSFORM (...) ] }
+SELECT [ hints , ... ] [ ALL | DISTINCT ] { [ [ named_expression | regex_column_names | star_clause ] [ , ... ] | TRANSFORM (...) ] }
```

Comment:
I wonder if we should do `star [ except_clause ]` instead of having except_clause inside the star_clause.

Reply:
I am referring to this doc https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-qry-select.html, as follows:
<img width="485" alt="image" src="https://github.com/user-attachments/assets/d882f6c4-2d9c-46fa-bf53-a59685e33a54">
Based on my understanding, I think `one possible reason` is that `EXCEPT` cannot exist independently of `STAR`, which seems to emphasize this relationship.